src/pages/StockPage.js

```diff
@@ -12,14 +12,12 @@ import { connect } from 'react-redux';
 import { getItemLayout, getPageDispatchers } from './dataTableUtilities';
 
 import { DataTable, DataTableHeaderRow, DataTableRow } from '../widgets/DataTable';
+import { ItemDetails } from '../widgets/bottomModals/ItemDetails';
+import { DataTablePageView, SearchBar } from '../widgets';
```

Comment:
confused why this is coming up in the diff https://github.com/openmsupply/mobile/blob/59db5205fb3b7574a4d60f8cad74cce237d6f3c3/src/pages/StockPage.js#L15-L16 !